src/js/components/stock-movement-wizard/fullOutbound/FullOutboundImport.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useMemo } from 'react';
+
+import FullOutboundImportConfirm from 'components/stock-movement-wizard/fullOutbound/FullOutboundImportConfirm';
+import FullOutboundImportDetails from 'components/stock-movement-wizard/fullOutbound/FullOutboundImportDetails';
+import WizardStepsV2 from 'components/wizard/v2/WizardStepsV2';
+import useWizard from 'hooks/useWizard';
+
+const FullOutboundImport = () => {
+  const FullOutboundImportStep = {
+    DETAILS: {
+      key: 'DETAILS',
+      // TODO: Make titles translatable in OBPIH-6329
+      title: 'Create',
+    },
+    CONFIRM: {
+      key: 'CONFIRM',
+      title: 'Confirm',
+    },
+  };
+  const steps = useMemo(() => [
```

Comment:
This feels a bit weird. Can we just have an object with full steps structure that would be transformed to list of values? Feels like this part could be simplified